hudi-spark/src/main/java/org/apache/hudi/keygen/parser/HoodieDateTimeParserImpl.java

```diff
@@ -58,7 +58,7 @@ public HoodieDateTimeParserImpl(TypedProperties config) {
     }
   }
 
-  private String getConfigInputDateFormatDelimiter() {
+  public String getConfigInputDateFormatDelimiter() {
```

Comment:
can we make this package-private?

Reply:
> can we make this package-private?

@pratyakshsharma thanks for your review.
I tried protected, it's not enough.